app/main/presenters/search_presenters.py

```diff
@@ -81,13 +81,16 @@ def _recursive_add_option_filters(question, options_list, filters_list):
             filters_list.append(presented_filter)
 
 
-def set_filter_states(filter_groups, request):
+def set_filter_states(filter_groups, base_url, clean_request_query_params):
     """Sets a flag on each filter to mark it as set or not"""
```

Comment:
This description could be updated to say it's not just marking a filter as set or not but also setting the links for those filters